jdi-light/src/main/java/com/epam/jdi/light/settings/ScreenSettings.java

```diff
@@ -4,15 +4,22 @@
 
 import java.io.File;
 
-import static com.epam.jdi.light.settings.JDISettings.*;
-import static com.epam.jdi.tools.PathUtils.*;
-import static com.epam.jdi.tools.Timer.*;
-import static java.lang.String.*;
-import static org.apache.commons.lang3.StringUtils.*;
+import static com.epam.jdi.light.settings.CommonSettings.getCommonSettings;
+import static com.epam.jdi.tools.PathUtils.mergePath;
+import static com.epam.jdi.tools.PathUtils.path;
+import static com.epam.jdi.tools.Timer.nowTime;
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.isBlank;
 
 public class ScreenSettings {
-    public String path = mergePath(COMMON.logsPath, "screens");
-    public String fileSuffix = "jpg";
+
+    public ScreenSettings() {
+        path = mergePath(getCommonSettings().logsPath, "screens");
+        fileSuffix = "jpg";
+    }
+
+    public String path;
+    public String fileSuffix;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)